public/app/plugins/datasource/jaeger/components/SearchForm.tsx

```diff
@@ -130,6 +131,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
             }
             isLoading={isLoading.operations}
             value={operationOptions?.find((v) => v.value === query.operation) || null}
+            placeholder="Select operation name"
```

Comment:
Can you change this to select an operation

Reply:
Done. https://github.com/grafana/grafana/pull/54026/commits/43fa1ac8e2ad12f054c5d07021b41222819c65ae